files/build_templates/docker_image_ctl.j2

```diff
@@ -418,13 +418,28 @@ start() {
     {%- if docker_container_name == "swss" %}
     # Generate the asic_table.json and peripheral_table.json
     if [ ! -f /etc/sonic/asic_table.json ] && [ -f /usr/share/sonic/templates/asic_table.j2 ]; then
-        sonic-cfggen -d -t /usr/share/sonic/templates/asic_table.j2 > /etc/sonic/asic_table.json
+        sonic-cfggen -d -t /usr/share/sonic/templates/asic_table.j2 > /etc/sonic/asic_table.json 2> errorlog.txt
+        if [[ $? -ne 0 ]] ; then
```

Comment:
>if [[ $? -ne 0 ]] ; then [](http://example.com/codeflow?start=8&length=24)

The error handling is useful and generic, we can separate into another PR.
This PR is WA, and will be revert in future.

Reply:
PR #14384 has been opened for the error-handling logic.
I'll adjust this PR once that one is merged.

Reply:
It is merged.

Reply:
Thanks. Conflict resolved.